### PR TITLE
Fix anchors in UG table of contents

### LINKED ISSUE #195: Update All Anchors in UG
Some anchors are broken because we change the headers and etc. We need to update all anchors especially in TBC. This can be done right after we finalise the content of our UG.

### PR DESCRIPTION
**Pull request by homingjun**
Fixes #195

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -279,7 +279,7 @@ The task Add parser has its priority changed to:
         High priority
 ```
 
-#### 3.3.5. Mark Task(s) as Complete
+#### 3.3.5. <a id="mark-task-as-complete">Mark Task(s) as Complete</a>
```
Comment (keke101) [inline docs/UserGuide.md]: Mj, can help to correct this?
* This header does not tally with the TBC
* Not consistent with other headers like "Add Tasks to the Sprint"

File: docs/UserGuide.md [hunk 2]
```diff
@@ -248,7 +248,7 @@ The details of the tasks are as follows:
     Completion: Completed
  ```
 
-#### 3.3.3. Delete Task(s)
+#### 3.3.3. <a id="delete-tasks">Delete Task(s)</a>
```
Comment (keke101) [inline docs/UserGuide.md]: Mj, can help make this consistent?
* This header does not tally with the TBC
* Not consistent with other headers like "Add Tasks to the Sprint"

File: docs/UserGuide.md [hunk 3]
```diff
@@ -216,7 +216,7 @@ Task successfully created.
         Task have yet to be assigned to anyone
 ```
 
-#### 3.3.2. View Task(s)
+#### 3.3.2. <a id="view-task">View Task(s)</a>
```
Comment (keke101) [inline docs/UserGuide.md]: Mj, can help make this consistent?
* This header does not tally with the TBC
* Not consistent with other headers like "Add Tasks to the Sprint"

**Review by keke101 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (homingjun) [reply docs/UserGuide.md]: Ok I've fixed the inconsistencies.

File: docs/UserGuide.md (hunk 2 above)
Comment (homingjun) [reply docs/UserGuide.md]: Ok I've fixed the inconsistencies.

File: docs/UserGuide.md (hunk 3 above)
Comment (homingjun) [reply docs/UserGuide.md]: Ok I've fixed the inconsistencies.

**Review by keke101 — APPROVED** [review]
Looks good to merge.

**Pull request merged**